aten/src/ATen/nn_parse.py

```diff
@@ -66,7 +66,7 @@ def map_to_th_type(t):
     def is_output_arg(arg_name, func_name):
         if arg_name == 'output' and 'updateOutput' in cname:
             return True
-        if name in {'gradInput', 'gradWeight', 'gradBias'}:
+        if name in {'gradInput', 'gradWeight', 'gradBias', 'gradGrid'}:
```

Comment:
What does this do?

Reply:
It's for binding arguments in THNN functions, specifically the updateGradInput functions. The two THNN functions added in the PRs are the first ones that have gradGrid, so I need to add this here too.